app/server/src/routes/space/space.validate.js

```diff
@@ -10,5 +10,8 @@ exports.validate = (method) => {
         body("image", "image does not exist").exists(),
       ];
     }
+    case "getSpaceDetail": {
+      return [param("id", "id does not exist").exists()];
```

Comment:
can you add ismongoid check to these

Reply:
Can you also add case for get all discussions

Reply:
Done!